src/System.Net.NameResolution/tests/PalTests/NameResolutionPalTests.cs

```diff
@@ -35,7 +35,6 @@ public void TryGetAddrInfo_LocalHost()
             Assert.NotNull(hostEntry.Aliases);
         }
 
-        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotArm64Process))] // [ActiveIssue(32797)]
```

Comment:
This is a test case inside a test project, we need [Fact]/[Theory] label to run the test. Here seems we remove the [ConditionalFact] without adding a new [Fact] label, and I don't think the test will actually run - or this is intentional? not running the test?

cc: @davidsh

Reply:
I agree.  And there is another test in this file also that now no longer runs.  It needs to be changed to "[Fact]" as well.